modules/edu.gemini.epics.acm/src/main/java/edu/gemini/epics/acm/CaApplySenderImpl.java

```diff
@@ -206,7 +206,7 @@ private WaitPreset(CaCommandMonitorImpl cm, CarStateGeneric carVal, Integer carC
         @Override
         public State onApplyValChange(Integer val) {
             if (val > 0) {
-                if (carClid != null && carClid.equals(val)) {
+                if (carVal != null && carClid != null && carClid.equals(val)) {
```

Comment:
I think you can replace `carClid != null && carClid.equals(val)` by `val.equals(carClid)`

Reply:
Doh!